src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -324,6 +287,9 @@ void Translation<Dim>::pup(PUP::er& p) {
   if (version >= 1) {
     p | f_of_r_;
     p | center_;
+  } else if (p.isUnpacking()) {
+    f_of_r_ = nullptr;
+    center_ = std::array<double, Dim>{0.};
```

Comment:
```cpp
center_ = make_array<Dim, double>(0.0);
```

Only specifying one `0.` will actually only initialize the first element for Dim = 2 and 3. (Technically the rest are default initialized to `0.` as well, but better to be pedantic here)

Also, to use this
```cpp
#include "Utilities/MakeArray.hpp
```